README.rst

```diff
@@ -112,6 +112,113 @@ By default both minidump-stackwalk and breakpad are built with PGO and an
+        "debug_id": <string>,                  // [[:xdigit:]]{33} | empty string
+        "end_addr": <string>,                  // 0x[[:xdigit:]]+
+        "filename": <string>,
+        "version": <string>,
+        "loaded_symbols": true,                // optional, if mdsw looked for the file and it does exist
+        "missing_symbols": true,               // optional, if mdsw looked for the file and it doesn't exist
+        "corrupt_symbols": true,               // optional, if mdsw found a file that has parse errors
+        "symbol_disk_cache_hit": <bool>,       // optional, whether or not the SYM file was fetched from disk cache
+        "symbols_fetch_time": <float>,         // optional, time in ms it took to fetch symbol file from url; omitted
+                                               // if the symbol file was in disk cache
+        "symbol_url": <string>                 // optional, url of symbol file
+      }
+    ],
+    "pid": <int>,                              // pid of crashed process
+    "thread_count": <int>,
+    "threads": [
+      // for i in range(thread_count)
+      {
+        "frame_count": <int>,
+        "frames": [
+          // for i in range(frame_count)
+          {
+            "frame": <int>,                    // frame index; 0-based
+            "module": <string>,                // optional
+            "function": <string>,              // optional
+            "file": <string>,                  // optional
+            "line": <int>,                     // optional
+            "offset": <string>,                // 0x[[:xdigit:]]+
+            "module_offset": <string>,         // 0x[[:xdigit:]]+ , optional
+            "function_offset": <string>        // 0x[[:xdigit:]]+ , optional
+            "missing_symbols": <bool>,
+            "trust": <string>,                 // none | scan | cfi_scan | frame_pointer | cfi | context | prewalked
+
+            // these next few are in the first frame only
+            "registers": {
+              // series of registers
+              <string>: <string>,              // name starts with r and then the name
+                                               // value is 0x[[:xdigit:]]
+            }
+          }
+        ]
+      }
+    ],
+    "tiny_block_size": <string>,
+
+    "unloaded_modules": [
+      // for i in range(unloaded_modules)
+      {
+        "base_addr": <string>,                 // 0x[[:xdigit:]]+
+        "code_id": <string>,
+        "end_addr": <string>,                  // 0x[[:xdigit:]]+
+        "filename": <string>
+      }
+    ],
+    
+    // this is a repeat of the crashing thread in the threads list, but the
+    // number of frames is truncated to 10
+    "crashing_thread": {
+      "threads_index": <int>,                  // index in threads for the crashing thread
+      "total_frames": <int>,                   // total frames listed here -- max 10
+      "frames": [
+        // for i in range(min(10, frame_count))
```

Comment:
I thought we removed this truncation because it was confusing?

Reply:
I forgot about that! Good catch!

https://github.com/mozilla-services/minidump-stackwalk/commit/5896847652369013057cf7d1733400f616b2f8aa